src/coreclr/binder/inc/assemblybindercommon.hpp

```diff
@@ -71,6 +76,15 @@ namespace BINDER_SPACE
 
         static HRESULT TranslatePEToArchitectureType(DWORD  *pdwPAFlags, PEKIND *PeKind);
 
+        static HRESULT DefaultBinderSetupContext(DefaultAssemblyBinder** ppTPABinder);
+
+        // TODO: The call indicates that this can come from a case where
```

Comment:
@VSadov is this TODO obsolete? If not, it would be nice to have an issue for it and reference it here.

Reply:
Possibly obsolete. CorLib may have NULL binder, but I am not sure if it goes through here. 

I will check on this `TODO` in later changes. 
I also think the statics on `AssemblyBinderCommon` may be moved to `AssemblyBinder`, but have not decided on that yet.

Reply:
I think some of this code will go to managed eventually, just want to clean it up a bit as-is beforehand. It seems like a good thing to do regardless.